utilities/rpp-performancetests/HIP/BatchPD_hip.cpp

```diff
@@ -1,824 +0,0 @@
-#include <stdio.h> 
```

Comment:
why all rpp-performancetests are deleted?

Reply:
Deleting the old folder now that the new one is fully functional and are only using that